runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the container process by the runtime.
+The runtime may leak additional file descriptors into the container process to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
```

Comment:
also, I think we should settle on something other than "container process" as the word container means too many things today. "application" seems reasonable to me.